cunumeric/array.py

```diff
@@ -2116,6 +2122,26 @@ def copy(self, order="C") -> ndarray:
         # We don't care about dimension order in cuNumeric
         return self.__copy__()
 
+    def cumsum(self, axis=None, dtype=None, out=None):
```

Comment:
please consider adding types to your methods. @bryevdv has been trying to do that (#387, #413), so let's help him with that.